l10n/en/products/vpn/landing.ftl

```diff
@@ -112,7 +112,7 @@ vpn-landing-faq-compatibility-question-desc-windows-v2 = <a href="{ $url }">{ -b
 
 # Variables:
 #   $url (url) - link to https://www.mozilla.org/products/vpn/desktop/mac/
-vpn-landing-faq-compatibility-question-desc-mac-v3 = <a href="{ $url }">{ -brand-name-mac }</a> (10.14 and up)
+vpn-landing-faq-compatibility-question-desc-mac-v3 = <a href="{ $url }">{ -brand-name-mac }</a> (10.15 and up)
```

Comment:
Hmmm. We should have avoided having the version number in the string. I don't think updating this will trigger new translations. We may have to manually update all of the translations with the new version number.

Reply:
Yeah, these strings are for the old version of the landing page, but we've had to update this version number more than once now, so even though it won't be around for a super long time, passing the version number as a variable is probably still worthwhile.

Reply:
Would a new id also be necessary here? i.e. `vpn-landing-faq-compatibility-question-desc-mac-v4`? Or do I add a variable to hold the min version string to all three variants of this string?

Reply:
@brizental yes, create a new string ID that accepts a variable (similar to the other strings you've already touched in other files here).

We'd still need to fallback to the old string for locales that have not yet translated the new string. See docs for some helpers here: https://bedrock.readthedocs.io/en/latest/l10n.html#the-ftl-helper-function

Reply:
Nice docs! 💯 I tried it, [53d89df](https://github.com/mozilla/bedrock/pull/13866/commits/53d89df7ebf4ea049dc316f169b8d48127df4a59). I forgot to undo the changes I made to the v3 string. I'll wait to know if the v4 string is correct and then I send a final commit to undo that.

Reply:
I haven't pulled the branch but it looks god to me! I think you're safe to undo the v3 string change, and then I'll double check things before merging. Thanks!

Reply:
Done!